### ISSUE TITLE
PHP 8.1/8.2 Compat

### ISSUE THREAD

**Issue by turtlepod** [10up/10up-experience#142]
### Describe the bug

Activating the plugin in site with PHP 8.2 trigger this deprecated notice:

```
Deprecated: strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated in /nas/content/live/devcic/wp-includes/functions.php on line 7127


Deprecated: str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /nas/content/live/devcic/wp-includes/functions.php on line 2182
```

### Steps to Reproduce

1. Install the plugin.
2. Make sure you use PHP 8.2
3. Make sure you set WP_DEBUG to true
4. Activate the plugin.
5. See the php notice 

### Screenshots, screen recording, code snippet

![image](https://github.com/10up/10up-experience/assets/1705549/5e6d89c3-30d7-4f86-be2c-7319fc278963)

### Environment information

Windows 11 Pro.
Chome (latest)

### WordPress information

Host: WP Engine
WP: Version 6.3.2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tylerb-10up** [comment]
This occurs on PHP 8.1 as well.

**Comment by TylerB24890** [comment]
This occurs on PHP 8.1 as well.

**Comment by turtlepod** [comment]
Thanks @TylerB24890 
Title updated.

**Comment by darylldoyle** [comment]
This is closed by #135
